check_deps/check_dependencies_v2.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python2
+
```

Comment:
The file has bad naming. There are files `check_dependencies.sh` and `check_dependencies_v2.py`. Seems like second one is more enhanced version of the first one, but it's not true.

Reply:
fixed